pkg/router/template/types.go

```diff
@@ -37,8 +35,15 @@ type ServiceAliasConfig struct {
 	// insecure connections to an edge-terminated route:
 	//   none (or disable), allow or redirect
 	InsecureEdgeTerminationPolicy routeapi.InsecureEdgeTerminationPolicyType
+
 	// Hash of the route name - used to obscure cookieId
 	RoutingKeyName string
+
+	// Annotations attached to this route
+	Annotations map[string]string
```

Comment:
This is fine to leave in, but I want the weights on the service, not the annotations.